index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/index.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <div class="head-title"> Weather Report</div>
+        <p>For the lovely city of <span id="city-display">Honolulu</span></p>
+    </header>
+    <main>
+        <section id="Temperature">
+            <h2> Temperature </h2>
+            <button id="up-arrow">⬆️</button>
+            <button id="down-arrow">⬇️</button>
+            <div id="temperature-number" class="color"></div>
+        </section>
+
+        <section id="Sky-Container">
+            <h3> Sky </h3>
```

Comment:
This section is a sibling of the temperature section, so the h-tag level would be the same.
```suggestion
            <h2> Sky </h2>
```